internal/core/workers/metricsreporter/metrics_reporter_worker_test.go

```diff
@@ -50,7 +49,7 @@ func TestMetricsReporterWorker_StartProduceMetrics(t *testing.T) {
 
 		mockCtl := gomock.NewController(t)
 		roomStorage := mock.NewMockRoomStorage(mockCtl)
-		instanceStorage := mockinstance.NewMockGameRoomInstanceStorage(mockCtl)
+		instanceStorage := mock.NewMockGameRoomInstanceStorage(mockCtl)
```

Comment:
What do you think to remove Mock from the function name, since it is duplicated inside the client call `mock.NewGameRoomInstanceStorage` will be simpler